docs/buildah-bud.md

```diff
@@ -355,6 +355,11 @@ Adjust the logging level up or down.  Valid option values range from -2 to 3,
 with 3 being roughly equivalent to using the global *--log-level=debug* option, and
 values below 0 omitting even error messages which accompany fatal errors.
 
+**--manifest** "manifest"
+
+Name of the manifest to which the image will be added. Creates the manifest if it does
```

Comment:
"manifest list", or "list", please.  Every image already has a manifest, and I worry that being imprecise about the terminology will foment confusion.

Reply:
Fixed